docs/releases/1.7.txt

```diff
@@ -637,7 +637,7 @@ Forms
   value.
 
 * :attr:`SelectDateWidget.months
-  <django.forms.extras.widgets.SelectDateWidget.months>` can be used to
+  <django.forms.SelectDateWidget.months>` can be used to
```

Comment:
Not too sure whichever one I prefer, in any case I'll wait for @timgraham's feedback regarding moving the deprecation to 1.8 before I amend the docs.